### ISSUE TITLE
HELP WANTED

### ISSUE THREAD

**Issue by dallasrlones** [emiliorizzo/vue-d3-network#16]
Is there a way to add multiple text fields for a node? In our case we're needing to label ports that the lines come in on.

Any help would be greatly appreciated!

**Comment by emiliorizzo** [comment]
No, you can't add multi line labels to node. But I can update the module, maybe next month
